addons/website/static/src/scss/website.scss

```diff
@@ -176,6 +176,16 @@ $-seen-urls: ();
     }
 }
 
+#oe_main_menu_navbar:not(.o_hidden) + #wrapwrap {
+    .o_header_affixed, .modal {
```

Comment:
Why no `:not(.s_popup_bottom)` like in the 14.2 commit you mention?

Reply:
Because it was a wrong fix due to the wrong height..